src/Program.cpp

```diff
@@ -13,7 +13,7 @@ void Lexer::Program::parse()
     Function main;
     main.name = "main";
     main.return_type = INT;
-    main.body.no_return = false;
```

Comment:
why

Reply:
before no_return was true for return and false for no_return so i reverse it to make it consistent against the variable name